0050-day-five/grooming/readme.md

```diff
@@ -0,0 +1,33 @@
+# Grooming
+
+## Content TODO's
+
+- Complete outline below
+- Provide guidance on criteria to determine if story can be estimated (see [definition of ready](../../team-agreements/definition-of-ready/readme.md))
+    - Examples of well defined acceptance criteria
+    - Can the story be tested as written
+    - Can it be completed within a sprint
+    - Is it dependent on other stories
+- Provide guidance on managing timeline for grooming
+  - time boxing story AC discussions
+- Provide guidance on estimate negotiation and how to resolve conflicts
+
+## Goals

```

Comment:
Important to note that you should only really spend time accurately estimating stories near the top of the backlog. As the project evolves, these estimates may be wildly different anyway:

From [Atlassian on Estimation](https://www.atlassian.com/agile/project-management/estimation):
> For items deeper in the backlog, give a rough estimate. By the time the team actually begins to work on those items, the requirements may change, and your application certainly will have changed. So prior estimates won’t be as accurate. Don’t waste time estimating work that is likely to shift. Just give the product owner a ballpark figure she can use to prioritize the product roadmap appropriately.

Reply:
Added bullet to define when to stop grooming.